wordpress-mobile/WordPress-iOS
# Auto saving drafts

Using a debouncer to call the saving method every 500ms (defined in constants)

Fixes # #10069

To test:
*Creating draft post*
1. Go to Blog posts
2. Go to Drafts tab
3. Start writing a blog post
4. Rerun XCode (as if the app quit suddenly)
5. Go to Blog Posts
Expected: Your post exists and saved as local with the latest changes.

*Editing post*
1. Go to Blog posts
2. Choose an exiting draft post
3. tap edit
4. start editing 
5.rerun XCode
Expected: Your posts updates exists as local with the latest changes.
